enhancements/declarative-index-config.md

```diff
@@ -69,11 +69,12 @@ A human consumable representation of packages in an index in the form of json/ya
 As a user with pull permission from the namespace an index is hosted in/as a component of OLM, I can query an index for a json/yaml representation of the packages of an index.
 The representation of individual packages will allow individual package owners to reason about/make changes to their individual packages in isolation.  
 
-`opm index inspect --image=docker.io/my-namespace/community-operators --output=json`
+`opm unpack docker.io/my-namespace/community-operators`
```

Comment:
I'm pretty sure `unpack` and `generate` is not done yet right?

Reply:
I had that same question when I looked earlier. Is the expectation that we're going to move the `opm alpha bundle unpack` to `opm unpack`, or is this going to be a new command entirely that's at the root?

Reply:
>I'm pretty sure unpack and generate is not done yet right?

No but this is probably the place to approve/discuss more on those commands? 

>  Is the expectation that we're going to move the opm alpha bundle unpack to opm unpack, or is this going to be a new command entirely that's at the root?

That's a good question, and an opportunity to all this out:  I think the plan is to ultimately go with `opm unpack <bundle-image|index-image>`

Reply:
This is an enhancement so it is perfectly to have some not-yet-implemented features. I want to make sure that the stories that are completed don't have any references to some not-yet-implemented commands. QE shouldn't try to attempt to verify things that are not done.

Reply:
That's a reasonable ask. JIRA is probably the place to interact with QE and make them aware of what's ready to be tested and what's yet to be implemented, while the enhancement has the record for all the planned/implemented work.